### PR TITLE
MyOrders Screen

### PR DESCRIPTION
**Pull request by mshago**

### CODE REVIEW

**Comment by mshago** [comment]
image

**Comment by mshago** [comment]
image

File: mobile/src/screens/myOrders/MyOrders.style.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {StyleSheet} from 'react-native';
+import {color, font, size} from '../../theme';
+
+export const styles = StyleSheet.create({
+  header: {
+    backgroundColor:color.white,
+    paddingHorizontal: size.rg,
+    paddingVertical:size.rg,
+    shadowColor: '#000',
```
Comment (Dulaya) [inline mobile/src/screens/myOrders/MyOrders.style.ts:9]: Can you do this? #000 --> color.black

File: mobile/src/screens/myOrders/MyOrders.type.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Dispatch, SetStateAction } from "react"
+import { TOrder } from "../../types/data"
+
+export type TMyOrdersScreen = {
+
+}
```
Comment (Dulaya) [inline mobile/src/screens/myOrders/MyOrders.type.ts:6]: export type TMyOrdersScreen = {
    navigation: TAppNavigation
};

File: mobile/src/screens/myOrders/MyOrders.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    delivered: true,
+    restaurantImage:
+      'https://logodownload.org/wp-content/uploads/2017/10/Starbucks-logo.png',
+    estimatedTime: '25 min',
+    total: '19.99',
+  },
+];
+
+export const MyOrdersScreen:FC<TMyOrdersScreen> = () => {
+
```
Comment (Dulaya) [inline mobile/src/screens/myOrders/MyOrders.tsx:58]: ({navigation})

// for props. You'll use this later

**Review by Dulaya — APPROVED** [review]

File: mobile/src/screens/myOrders/MyOrders.tsx (hunk 3 above)
Comment (mshago) [reply mobile/src/screens/myOrders/MyOrders.tsx:58]: done

File: mobile/src/screens/myOrders/MyOrders.type.ts (hunk 2 above)
Comment (mshago) [reply mobile/src/screens/myOrders/MyOrders.type.ts:7]: done

File: mobile/src/screens/myOrders/MyOrders.style.ts (hunk 1 above)
Comment (mshago) [reply mobile/src/screens/myOrders/MyOrders.style.ts]: done

**Review by raynesmike — APPROVED** [review]
I approve this

**Review by carizzuti — APPROVED** [review]

**Pull request merged**